pkg/build/api/types.go

```diff
@@ -911,6 +915,9 @@ type GenericWebHookEvent struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
+
+	// StrategyOptions contains additional strategy-specific options for the build
+	StrategyOptions *BuildStrategyOptions
```

Comment:
@smarterclayton bump.  i'm open to other naming suggestions but i don't want to just call it DockerStrategy.